README.md

```diff
@@ -14,6 +14,8 @@ We designed Notary for Enterprise environments where users need to securely mana
 - **Simple UI**: A simple and intuitive web interface for managing certificates.
 - **Extensive HTTP API**: Accomplish all the tasks you can do in the UI via the HTTP API.
 - **Metrics**: Monitor the state of your certificates and the health of your Notary instance with Prometheus metrics.
+- **Encryption at Rest**: Sensitive data like private keys is encrypted in the database.
+- **Encryption Backend Support**: To increase security an external encryption backend can be configured to encrypt Notary's encryption key.
```

Comment:
Please use the same content as in the docs index (i.e. only the point about hsm)

Reply:
:white_check_mark: